### PR TITLE
Updated the left arrow

### LINKED ISSUE #3682: [Homepage] News section: left and right arrows need improvement
### Description
See the screenshot.

### Expected Behavior
Arrow should not be cutoff.

### Screenshots
Screenshot 2023-02-02 at 5 01 34 PM

### Environment:
- Host OS: macOS
- Browser: Safari

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ShuvamChatterjee28**
Signed-off-by: Deb Chatterjee <38078961+ShuvamChatterjee28@users.noreply.github.com>

**Description**
The left and right cut-off feature is fixed now. Below is the screenshot from my safari.
Screenshot 2023-02-12 at 11 19 08 PM

This PR fixes #3682 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 82ab9e438b51132ccd429e510b61a00a1d9bc682 at: https://63e92afb3d8d352ec3e7efb3--layer5.netlify.app

**Comment by ShuvamChatterjee28** [comment]
@leecalcote you can review it.

**Comment by ShuvamChatterjee28** [comment]
@Nikhil-Ladha can you please look into this PR?

**Comment by l5io** [comment]
🚀 Preview for commit 7a8e47eb2ad0825ca916d5b3bb0d14925c4e5a6a at: https://63e9bea6744a8f1347e25c89--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
Have you cross-checked the arrow fix in Safari?

**Comment by Nikhil-Ladha** [comment]
Some more changes would be needed for the banner issue.
I am pushing those changes to the other PR that Oghene opened. Please limit the changes in this PR to only the navigation arrows.

**Comment by ShuvamChatterjee28** [comment]
> Have you cross-checked the arrow fix in Safari?

Yes, I have checked and the above screenshot is taken from safari browser from my Mac.

**Comment by Nikhil-Ladha** [comment]
> > Have you cross-checked the arrow fix in Safari?
> 
> Yes, I have checked and the above screenshot is taken from safari browser from my Mac.

Ok, 2 things in that case:
1. The left arrow shouldn't have any hover effect when not active.
2. The right arrow is still being cutoff a bit at the edge.

Please address the above 2 issues.

**Comment by ShuvamChatterjee28** [comment]
Okay @Nikhil-Ladha, I will do that. I will close this one, as I messed up somehow. Can you please tell me what to do for the banner changes?

**Comment by ShuvamChatterjee28** [comment]
I think it is fixed.

**Comment by l5io** [comment]
🚀 Preview for commit 09204365d5130fb8b4da084560d6ea02ec3fd8cf at: https://63e9d5fe16ba5423dbe6629a--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
Please address these issues:

- The left arrow shouldn't have any hover effect when not active.
- The left and right arrows are not equidistant from the slider.
- The right arrow is still being cut off a bit at the edge.

Please address the above 2 issues.

**Comment by ShuvamChatterjee28** [comment]
Yes I am on it. But how to check if the arrow is active? @Nikhil-Ladha any lead?

**Comment by Nikhil-Ladha** [comment]
When the arrow is disabled it has a `slick-disabled` class added to it.

**Comment by ShuvamChatterjee28** [comment]
Done @Nikhil-Ladha, you can have a look.

**Comment by l5io** [comment]
🚀 Preview for commit ab1c78089cdf477bf9009af20dcb3b5ecc09f128 at: https://63e9ed868dc47b2ed0ae35e9--layer5.netlify.app

File: src/sections/Home/So-Special-Section/so-special-style.js [hunk 1]
```diff
@@ -23,7 +23,11 @@ const SoSpecialWrapper = styled.div`
     }
     
     .slick-next{
-        right: -2.5rem;
+        right: -2rem;
+    }
```
Comment (Nikhil-Ladha) [inline src/sections/Home/So-Special-Section/so-special-style.js:27]: I see where the issue was, let's revert it to `2.5rem` only.

**Review by Nikhil-Ladha — COMMENTED** [review]
Can you please share the latest screenshot from Safari?
After incorporating the last comment.

**Comment by ShuvamChatterjee28** [comment]
> Can you please share the latest screenshot from Safari?
> After incorporating the last comment.

Screenshot 2023-02-13 at 3 06 54 PM

Here is the last screenshot after changing it back to `-2.5rem`. Also it is taken from Safari (Mac).

**Comment by l5io** [comment]
🚀 Preview for commit e8452ca9aa1176bdfe8c258747d80e75308e1b96 at: https://63ea095afb18ad4f355c4d19--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**